js_modules/dagit/packages/core/src/assets/AutoMaterializePolicyPage/AssetAutomaterializePolicyPage.tsx

```diff
@@ -24,92 +99,231 @@ export const AssetAutomaterializePolicyPage = () => {
-        const date = dayjs.unix(evaluation.date);
-        const formattedDate = date.format('lll');
-        return (
-          <CenterAlignedRow
-            flex={{justifyContent: 'space-between'}}
-            key={evaluation.date + ''}
-            padding={{horizontal: 24, vertical: 8}}
-            border={{side: 'bottom', width: 1, color: Colors.KeylineGray}}
-          >
-            {formattedDate}
-            {evaluation.materialized ? (
-              <Icon name="auto_materialize_policy" size={24} />
-            ) : (
-              <div
-                style={{borderRadius: '50%', height: '10px', width: '10px', color: Colors.Yellow50}}
-              />
-            )}
-          </CenterAlignedRow>
-        );
-      })}
+    <Box flex={{direction: 'column', grow: 1}} style={{overflowY: 'auto'}}>
+      <Box style={{flex: 1, minHeight: 0, overflowY: 'auto'}} flex={{grow: 1, direction: 'column'}}>
+        {evaluations.map((evaluation) => {
+          const isSelected = selectedEvaluation === evaluation;
+          return (
+            <EvaluationRow
+              flex={{justifyContent: 'space-between'}}
+              key={evaluation.evaluationId}
+              padding={{horizontal: 24, vertical: 8}}
+              border={{side: 'bottom', width: 1, color: Colors.KeylineGray}}
+              onClick={() => {
+                onSelectEvaluation(evaluation);
+              }}
+              $selected={isSelected}
+            >
+              <TimestampDisplay timestamp={evaluation.timestamp} />
+              {evaluation.numRequested ? (
+                <Icon name="auto_materialize_policy" size={24} />
+              ) : evaluation.numSkipped ? (
+                <div
+                  style={{
+                    borderRadius: '50%',
+                    height: '10px',
+                    width: '10px',
+                    color: Colors.Yellow500,
+                  }}
+                />
+              ) : (
+                <div
+                  style={{
+                    borderRadius: '50%',
+                    height: '10px',
+                    width: '10px',
+                    color: Colors.Yellow50,
+                  }}
+                />
+              )}
+            </EvaluationRow>
+          );
+        })}
+      </Box>
+      {queryResult.data?.autoMaterializeAssetEvaluations?.length ? (
+        <PaginationWrapper>
+          <CursorPaginationControls {...paginationProps} />
+        </PaginationWrapper>
+      ) : null}
     </Box>
   );
-};
+}
+
+const RightPanel = ({
+  assetKey,
+  setMaxMaterializationsPerMinute,
+}: {
+  assetKey: Omit<AssetKey, '__typename'>;
+  setMaxMaterializationsPerMinute: (max: number) => void;
+}) => {
+  const queryResult = useQuery<GetPolicyInfoQuery, GetPolicyInfoQueryVariables>(
+    GET_POLICY_INFO_QUERY,
+    {
+      variables: {
+        assetKey,
+      },
+    },
+  );
+  useQueryRefreshAtInterval(queryResult, FIFTEEN_SECONDS);
```

Comment:
I think this query is super lightweight so this is fine, but in most other places in the app we don't live reload the definition of the asset node, just the current state of it. (I think this is the only page where something like the automaterializePolicy autoreloads)